tests/bud/buildkit-mount-from/Dockermultistagefrom

```diff
@@ -0,0 +1,6 @@
+FROM quay.io/jitesoft/alpine as builder
```

Comment:
Why the sudden change in base image?

Reply:
Fixed in latest commit, Actually docker was rate-limiting my pulls on local :sweat_smile:  so had to change for local and i committed it by mistake.